### PR TITLE
Release coq-itree 5.1.2

### PR DESCRIPTION
**Pull request by Lysxia**

### CODE REVIEW

File: released/packages/coq-itree/coq-itree.5.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "Gregory Malecha"
+  "Steve Zdancewic"
+  "Benjamin Pierce"
+]
+license: "MIT"
+homepage: "https://github.com/DeepSpec/InteractionTrees"
+bug-reports: "https://github.com/DeepSpec/InteractionTrees/issues"
+dev-repo: "git+https://github.com/DeepSpec/InteractionTrees.git"
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (palmskog) [inline released/packages/coq-itree/coq-itree.5.1.2/opam:18]:
```suggestion
  ["dune" "subst"] {dev}
```
opam maintainers said this should always be `dev` these days.

**Comment by liyishuai** [comment]
Maybe worth adding `coq-itree-extra.5.1.2` as well? @Lysxia

**Comment by Lysxia** [comment]
There wasn't any change to that part so it should still build fine.

**Comment by liyishuai** [comment]
Not so fine: https://github.com/DeepSpec/InteractionTrees/commit/466800c19ab7ab84ab907617586b1039f2b8a9f6#diff-304307cde00c60adfc92e07582180ff44074416fdaef01eb7076ab2d1af73ad6
Blocking coq-community/coq-ext-lib#41

**Comment by Lysxia** [comment]
Oh I see I had forgotten to release 5.1.1 before. To be fixed by #2804

**Pull request merged**